ipynb/scripts/expand.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+"""
+This script is courtesy of Dr. Daniel McDonald, originally found in:
```

Comment:
maybe link [here](https://github.com/wasade/reimagined-fiesta) instead of barnacle?